nav2_regulated_pure_pursuit_controller/src/regulated_pure_pursuit_controller.cpp

```diff
@@ -383,10 +389,12 @@ bool RegulatedPurePursuitController::isCollisionImminent(
   // odom frame and the carrot_pose is in robot base frame.
 
   // check current point is OK
-  if (inCollision(robot_pose.pose.position.x, robot_pose.pose.position.y)) {
+  if (inCollision(
+      robot_pose.pose.position.x, robot_pose.pose.position.y,
+      tf2::getYaw(robot_pose.pose.orientation)))
+  {
     return true;
   }
-
```

Comment:
Readd the 2 spaces (here and L413) please, just a styling thing so I can parse this easier in the future, nothing technically wrong with it.

Reply:
where to add it .  any change made on it makes uncrustify test fail

Reply:
L389, where this comment is located. If you look at the diff, there was a space after the `}` and before the comment.